twitterMini/Twitter.swift

```diff
@@ -0,0 +1,130 @@
+//
+//  Status.swift
+//  twitterMini
+//
+//  Created by Tom O'Malley on 5/2/16.
+//  Copyright © 2016 intrepid. All rights reserved.
+//
+
+import Foundation
+import OAuthSwift
+
+// MARK: Status/User Objects
+
+struct TwitterStatus {
+    let createdAt: String
+    let text: String
+    let user: TwitterUser
+    let likes: String
+    let retweets: String
+    //TODO: deal with media (["extendedEntities"]["media"]["media_url_https"])
+}
+
+struct TwitterUser {
+    let screenName: String
+    let name: String
+    let profileImageURL: String
+}
+
+// MARK: API Client
+
+class TwitterAPIClient {
+    static let sharedClient = TwitterAPIClient()
+    
+    let twitterBaseURL = "https://api.twitter.com/1.1/"
+    let callback = "twittestIntrepid://"
+    
+    let oauthManager = OAuth1Swift(
+        consumerKey:    "CbE3BbqhqwxNbRdGCP5y2gDU2",
+        consumerSecret: "PxQy9075tg4QSZsLZNX3wwXEkG5fyuhtrAlNjM868sYRi6BCU8",
+        requestTokenUrl: "https://api.twitter.com/oauth/request_token",
+        authorizeUrl:    "https://api.twitter.com/oauth/authorize",
+        accessTokenUrl:  "https://api.twitter.com/oauth/access_token"
+    )
+    private var idForLoggedInUser: String?
+    
+    func authorizeTwitter(completion: (success: Bool) -> Void){
```

Comment:
Spacing, should be : `func authorizeTwitter(completion: (success: Bool) -> Void) {`